tests/test_FindTest.py

```diff
@@ -314,12 +314,10 @@ def test_find_citations(self):
              [],),
             ('lorem 111 N. W. 12th St.',
              [],),
-            # Test that the tokenizer handles whitespace well. In the past, the
-            # capital letter P in 5243-P matched the abbreviation for the Pacific
-            # reporter ("P"), and the tokenizing would be wrong.
-            ('Failed to recognize 1993 Ct. Sup. 5243-P',
+            # Test Conn. Super. Ct. regex variation.
+            ('Failed to recognize 1993 Conn. Super. Ct. 5243-P',
              [case_citation(3, volume='1993', reporter='Conn. Super. Ct.',
-                            reporter_found='Ct. Sup.', page='5243-P')]),
+                            page='5243-P')]),
```

Comment:
Would `1993 Ct. Sup. 5243-P` no longer be extracted? (Is `5243-P` only a valid page number for `Conn. Super. Ct.` and not `Ct. Sup.`?

Reply:
Like `160407WC`, `1993 Ct. Sup. 5243-P` should work with the version of reporters-db on master -- [here's the PR](https://github.com/freelawproject/reporters-db/pull/46).